Specs/Number/French/NumberRangeModel

```diff
@@ -0,0 +1,1359 @@
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "environ 500 ou plus",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[500,)"
+        },
+        "Start": 10,
+        "End": 29
+      }
+    ]
+  },
+  {
+    "Input": "Plus de la moitié des gens sont venus ici.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "Plus de la moitié",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(0.5,)"
+        },
+        "Start": 0,
+        "End": 17
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont plus petits ou égaux à 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "plus petits ou égaux à 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 64
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont inférieurs ou égaux à 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "inférieurs ou égaux à 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 63
+      }
+    ]
+  },
+  {
+    "Input": "Il y en a 500 ou moins dans ces produits.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "500 ou moins",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,500]"
+        },
+        "Start": 10,
+        "End": 22
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont < = 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "< = 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 45
+      }
+    ]
+  },
+  {
+    "Input": "Sa taille est en dessuous de 170.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "en dessuous de 170",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,170)"
+        },
+        "Start": 14,
+        "End": 32
+      }
+    ]
+  },
+  {
+    "Input": "Sa taille est inférieure à 170.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "inférieure à 170",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,170)"
+        },
+        "Start": 14,
+        "End": 30
+      }
+    ]
+  },
+  {
+    "Input": "Moins d’un millier de pandas géants vivent encore dans la nature.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "Moins d’un millier de",
```

Comment:
"de" should not be tagged.

Reply:
French translation should be consistent with original English data. In this case, if there is an "of" in an English output text, the French translation will also contain "de", vice versa.